feature-libs/product-configurator/rulebased/components/add-to-cart-button/configurator-add-to-cart-button.component.html

```diff
@@ -2,7 +2,12 @@
   <ng-container *ngIf="!container.routerData.displayOnly; else displayOnly">
     <div class="cx-add-to-cart-btn-container">
       <button
-        class="cx-btn btn btn-block btn-primary cx-add-to-cart-btn"
+        class="
+          cx-btn
+          btn btn-block btn-primary
+          cx-add-to-cart-btn
+          floating-button
```

Comment:
Do you use the floating-button class?
I can not find it anywhere else.

Reply:
done